packages/store/src/organizationSlice/thunks.ts

```diff
@@ -395,20 +498,20 @@ export const deleteSsoConfig = createAsyncThunk(`${sliceName}/deleteSsoConfig`,
     })
 );
 
-export const getSsoConfigById = createAsyncThunk(`${sliceName}/getSsoConfigById`, (config, { dispatch }) =>
-  Api.get(`${ssoIdpApiUrlv1}/${config.id}`)
+export const getSsoConfigById = createAppAsyncThunk(`${sliceName}/getSsoConfigById`, (ssoConfig: SSOConfig, { dispatch }) =>
+  Api.get(`${ssoIdpApiUrlv1}/${ssoConfig.id}`)
     .catch(err => dispatch(ssoConfigActionErrorHandler(err, 'read')))
     .then(({ data, headers }) => {
       const sso = Object.values(SSO_TYPES).find(({ contentType }) => contentType === headers['content-type']);
-      return sso ? Promise.resolve({ ...config, config: data, type: sso.id }) : Promise.reject('Unsupported SSO config content type.');
+      return sso ? Promise.resolve({ ...ssoConfig, config: data, type: sso.id }) : Promise.reject('Unsupported SSO config content type.');
     })
 );
 
-export const getSsoConfigs = createAsyncThunk(`${sliceName}/getSsoConfigs`, (_, { dispatch }) =>
-  Api.get(ssoIdpApiUrlv1)
+export const getSsoConfigs = createAppAsyncThunk(`${sliceName}/getSsoConfigs`, (_, { dispatch }) =>
+  Api.get<SSOConfig>(ssoIdpApiUrlv1)
     .catch(err => dispatch(ssoConfigActionErrorHandler(err, 'readMultiple')))
     .then(({ data }) =>
-      Promise.all(data.map(config => dispatch(getSsoConfigById(config)).unwrap()))
+      Promise.all(data.map((ssoConfig: SSOConfig) => dispatch(getSsoConfigById(ssoConfig)).unwrap()))
```

Comment:
Why the rename? as in: is there any chance we're talking about something else than an SSOConfig or just personal preference?

Reply:
Thanks for the question! I renamed it to `ssoConfig` for clarity, as it better reflects that we are talking about `ssoConfig` and not the `config` property of the `SSOConfig` type. The rename was made to reduce cognitive load and remove ambiguity. I’m happy to revert if the name config is more consistent, let me know